pkg/api/defaults-kubelet.go

```diff
@@ -77,6 +77,9 @@ func (cs *ContainerService) setKubeletConfig(isUpgrade bool) {
 	staticWindowsKubeletConfig["--image-pull-progress-deadline"] = "20m"
 	staticWindowsKubeletConfig["--resolv-conf"] = "\"\"\"\""
 	staticWindowsKubeletConfig["--eviction-hard"] = "\"\"\"\""
+	if to.Bool(o.KubernetesConfig.UseCloudControllerManager) {
```

Comment:
I don't think this is necessary. Windows nodes are never used as control plane nodes, and worker node kubelet runtimes don't have a functional "external cloud-provider" context.